src/components/swimlanes/swimLanes.tsx

```diff
@@ -29,10 +29,14 @@ const SwimLanes = ({ lanesWithNumItems }) => {
           type="standalone"
           href={`${DC_URL}/collections/lane/${lane.slug}`}
           aria-label={`See more ${lane.title.toLowerCase()}`}
+          hasVisitedState={false}
           sx={{
             display: { sm: "none", md: "inline" },
-            fontWeight: "500",
+            color: "ui.blue",
+            fontWeight: "500 !important",
             alignItems: "center",
+            // marginBottom: "1px",
+            _hover: { textDecoration: "underline 1px dotted !important" },
```

Comment:
This is so odd. This should the default but it's not working. I tried making this `sx={{}}` to test and it throws the styles off. This is a bug in the DS but luckily you basically don't need any of this code.

Update this by:
- add `fontWeight="medium"` as a style prop
- remove the entire `sx` block.

You should have the desired result. I'll make a bug ticket for the empty sx value affecting styles though.

Reply:
this fixes the text color and weight but doesn't fix the alignment issue of the text and the arrow.

Reply:
Is the default alignment considered "wrong"?

Reply:
i believe so, I'm just going off what is in prod dc but I'll ask lily

Reply:
got confirmation from lily that the default alignment is incorrect

Reply:
Okay, then I'd prefer if this is fixed in the DS so you don't have to add an override. Any thoughts @bigfishdesign13 ?